src/ui_fsmenu/CMakeLists.txt

```diff
@@ -97,7 +97,16 @@ wl_library(ui_fsmenu_gameloading
     launch_spg.h
     loadgame.cc
     loadgame.h
+    mapdetailsbox.cc
+    mapdetailsbox.h
+	multiplayersetupgroup.cc
```

Comment:
We are using space here - are you using tabs?

Reply:
yes, sorry. fixed.